app/shared/ui-foundation/src/commonMain/kotlin/platform/PlatformWindow.kt

```diff
@@ -1,6 +1,11 @@
 package me.him188.ani.app.platform
 
 /**
- * PC 上的 Window. Android 上没有
+ * 公共的 Window 抽象
+ * @property isUndecoratedFullscreen 当前窗口是否处于全屏模式
+ * @property isLandscape 当前窗口是否处于横屏模式，PC 上一定处于横屏模式，Android 从 configuration 里读
  */
-expect class PlatformWindow
+expect class PlatformWindow {
+    val isUndecoratedFullscreen: Boolean
+    val isLandscape: Boolean
```

Comment:
```suggestion
    val deviceOrientation: DeviceOrientation
```

```kotlin
enum class DeviceOrientation {
    PORTRAIT,
    LANDSCAPE,
}
```

enum 要比 boolean 更 readable, 使用 `deviceOrientation` 也可以突出这个东西是设备的旋转方向, 区分页面自己的全屏模式之类的.